src/helper/accepts/accepts.ts

```diff
@@ -27,16 +27,24 @@ export interface acceptsOptions extends acceptsConfig {
 
 export const parseAccept = (acceptHeader: string): Accept[] => {
   // Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8
-  const accepts = acceptHeader.split(',') // ['text/html', 'application/xhtml+xml', 'application/xml;q=0.9', 'image/webp', '*/*;q=0.8']
+  const accepts = acceptHeader.split(',')
```

Comment:
Hi, @Jayllyz 
Thanks for the PR. Very good fix!

I would like to see the comments here dropped, if it is an unintentional deletion, please put it back!

Reply:
Done!